src/articles/2020/05/05/jamstack-is-fast-only-if-you-make-it-so/index.md

```diff
@@ -16,60 +16,60 @@ Eleventy is very different, as Zach Leatherman reminds us in [Eleventy’s New P
-- Everyone should know that [Aaron Parecki](https://aaronparecki.com/) provides the awesome webmention.io service **for free**, and most Webmention users seem to use it nowadays, so being nice with its API feels better
+- The performance for the users is much better, with HTML already computed on the server and statically served.
+- Much fewer API calls, requiring much less computing time and power.
+- Everyone should know that [Aaron Parecki](https://aaronparecki.com/) provides the awesome webmention.io service **for free**, and most Webmention users seem to use it nowadays, so being nice with its API feels better.
 
-# Enhance client side, if really needed
+# Enhance client-side, if really needed
 
-If you know you receive a lot of very useful webmentions that you have to show to your visitors, you can enhance the server side generated list with a bit of client side.
+If you know you receive a lot of very useful webmentions that you have to show to your visitors, you can enhance the server-side generated list with a bit of client-side.
 
 But remember every JavaScript added to the page has a cost, so the few additional webmentions have to be really useful.
 
 So, instead of doing this for every page view, at least:
 
-First, try to **wait for some time after the site build** before making client side API calls. Keep the build timestamp available to client side JavaScript, and wait for an hour, a day, or more, depending on the frequency of webmentions. You could even use the page's "age" to query wbemention.io less for older content that probably receive less webmentions, as [Aaron Gustafson did even for server side call in his Jekyll plugin](https://aarongustafson.github.io/jekyll-webmention_io/performance-tuning).
+First, try to **wait for some time after the site build** before making client-side API calls. Keep the build timestamp available to client-side JavaScript, and wait for an hour, a day, or more, depending on the frequency of webmentions. You could even use the page's "age" to query webmention.io less for older content that probably receives less webmentions, as [Aaron Gustafson did even for server-side call in his Jekyll plugin](https://aarongustafson.github.io/jekyll-webmention_io/performance-tuning).
 
 Then, **keep track of a user's calls to the API**, in localStorage or IndexedDB, so that you don't make these calls again a short while after. You could even use a Service Worker to cache requests and their timestamp.
 
-# Client side only API calls sometimes make more sense
+# Client-side only API calls sometimes make more sense
 
-I agree Webmentions are not the most complex use case to explain that you should as much as possible call APIs from the server at build time instead of from the client:
+I agree Webmentions are not the most complex use case to explain that you should use as much as possible call APIs from the server at build time instead of from the client:
```

Comment:
```suggestion
I agree Webmentions are not the most complex use case to explain that most of the time you should call APIs from the server at build time rather than from the client:
```